prog/vm/github_runner.rb

```diff
@@ -52,12 +52,11 @@ def pick_vm
         billing_rate_id: BillingRate.from_resource_properties("IPAddress", "IPv4", picked_vm.location)["id"],
         amount: 1
       )
-
-      puts "#{project} Pool is used for #{label}"
+      Clog.emit("Pool is used") { {github_runner: {ubid: github_runner.ubid, label: github_runner.label, repository_name: github_runner.repository_name}} }
```

Comment:
Don't you get the ubid for free in the automatic thread field? Is that not good enough? (perhaps we should name it "ubid" or "subject"

Reply:
Yeah, actually it's enough. Let me remove it

Reply:
I also added `cores` to allow us to view the requested core count in graphs. The requests for `standard-16` and `standard-2` don't affect the total capacity in the same way.